Microsoft.Toolkit.Uwp.UI.Controls/TextBoxMask/TextBoxMask.cs

```diff
@@ -119,10 +120,38 @@ private static void Textbox_Loaded(object sender, RoutedEventArgs e)
             textbox.TextChanging += Textbox_TextChanging;
             textbox.SelectionChanged += Textbox_SelectionChanged;
             textbox.Paste += Textbox_Paste;
+            textbox.GotFocus += Textbox_GotFocus;
             textbox.SetValue(OldTextProperty, textbox.Text);
             textbox.SelectionStart = 0;
         }
 
+        private static void Textbox_GotFocus(object sender, RoutedEventArgs e)
+        {
+            var textbox = sender as TextBox;
```

Comment:
I'd suggest doing a single `if (textbox == null) return;` after this instead of all these null checks on every reference to `textbox`.

Having said that, this is a private method only used with the `TextBox.GotFocus` event, so the `sender` should always be set with the current instance of the TextBox control!